### PR TITLE
Wrap a `returnExports` UMD pattern

### PR DESCRIPTION
**Pull request by kenvunz**

### CODE REVIEW

File: subclass-error.js [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+        // AMD. Register as an anonymous module.
+        define([], factory);
+    } else if (typeof exports === 'object') {
+        // Node. Does not work with strict CommonJS, but
+        // only CommonJS-like environments that support module.exports,
+        // like Node.
+        module.exports = factory();
+    } else {
+        // Browser globals (root is window)
+        root.returnExports = factory();
```
Comment (dasilvacontin) [inline subclass-error.js]: Why `root.returnExports` and not something like `root.SubclassError`?

**Comment by dasilvacontin** [comment]
Thanks for the PR!

I made a comment on one of the lines. Please squash commits after your changes (if any), thanks! :)

File: subclass-error.js (hunk 1 above)
Comment (kenvunz) [inline subclass-error.js]: duh, of course! that would not be very cool to have a `returnExports` as the global variable :smile: 

i'll get it sorted.

**Comment by kenvunz** [comment]
Hey @dasilvacontin just amended the fix as you suggested. Not sure if did it correctly tho, let me know, a bit new to this whole Pull Request things

Cheers
Ken

**Comment by dasilvacontin** [comment]
You did it correctly, thanks again for the PR. :)

**Pull request merged**